tests/ui_tools/test_boxes.py

```diff
@@ -251,6 +251,34 @@ def test_generic_autocomplete_emojis(self, write_box, text,
         typeahead_string = write_box.generic_autocomplete(text, state)
         assert typeahead_string == required_typeahead
 
+    @pytest.mark.parametrize(['suggestions', 'state', 'expected_state',
+                              'expected_typeahead', 'is_truncated'], [
+      (['zero', 'one', 'two'], 1, 1, '*one*', False),
+      (['zero', 'one', 'two'] * 4, 1, 1, '*one*', True),
+      (['zero', 'one', 'two'], None, None, None, False),
+      (['zero', 'one', 'two'], 5, None, None, False),
+      (['zero', 'one', 'two'], -5, None, None, False),
+     ], ids=[
+        'less_than_10_typeaheads',
```

Comment:
minor: These are 'countable', so 'fewer' would be more accurate than 'less'

Reply:
Fixed